src/Packaging.cmake

```diff
@@ -0,0 +1,179 @@
+
+find_package(Qt5Core REQUIRED)
+get_target_property(_qmake_executable Qt5::qmake IMPORTED_LOCATION)
+get_filename_component(_qt_bin_dir "${_qmake_executable}" DIRECTORY)
+find_program(WINDEPLOYQT_EXECUTABLE windeployqt HINTS "${_qt_bin_dir}")
+find_program(LINUXDEPLOYQT_EXECUTABLE linuxdeployqt HINTS "${_qt_bin_dir}")
```

Comment:
Here I assumes `linuxdeployqt` is accecible as a name `linuxdeployqt` here. There may be a `linuxdeployqt-continuous-x86_64.AppImage`